main.lua

```diff
@@ -231,6 +231,7 @@ function EID:printDescription(desc)
 	--Display Itemname
 	if EID.Config["ShowItemName"] then
 		local curName = desc.Name
+		local quality = EID.itemWeightsLookup[tonumber(desc.ID)]
```

Comment:
that will throw errors on modded items. Lookup table is fine, but should be avoided when modded items are potential targets. So please read the quality from the game directly: https://wofsauge.github.io/IsaacDocs/rep/ItemConfig_Item/#quality